gym/wrappers/autoreset.py

```diff
@@ -39,6 +41,13 @@ class AutoResetWrapper(gym.Wrapper):
     """
 
     def step(self, action):
+        """Steps through the environment with action and resets the environment `if done=False`.
+
+        Args:
+            action: The action to take
+
+        Returns: The autoreset environment :meth:`step`
```

Comment:
I think that the "Returns" section is a little confusing since it seems to imply that `step()` returns a method

Reply:
Fixed